static/js/app.js

```diff
@@ -402,7 +402,9 @@ function runPythonProgram(code, hasTurtle, cb) {
     read: builtinRead,
     inputfun: inputFromInlineModal,
     inputfunTakesPrompt: true,
-    __future__: Sk.python3
+    __future__: Sk.python3,
+    // Give up after three seconds of execution, there might be an infinite loop.
+    execLimit: 3000
```

Comment:
Can we put this value in a function that depends on level? For now the function can just be any level returns 3000 but that gives us the flexibility to change it depending on level.